### ISSUE TITLE
Missing Gnome pie indicator icon

### ISSUE THREAD

**Issue by bil-elmoussaoui** [numixproject/numix-icon-theme#498]
Hello, 
As said in the title a Gnome pie indicator is missed! and i guess this is a wont fix but i just wanted to be sure.
the icon is 
```
gnome-pie-symbolic.svg
```
and the directory is : 
```
/usr/share/icons/hicolor/scalable/apps
```
the default icon : 
![screenshot from 2015-05-24 02-44-51](https://cloud.githubusercontent.com/assets/7660997/7786234/e12cea9c-01be-11e5-8bb6-8cfbdafdf058.png)
Ps: this is a screenshot from an image viewer! 

I think the best thing is to create a gnome-pie-indicator in status folder and report this to gnome pie devs.

**Comment by dirtydancing** [comment]
This is not necessarily a bug in gnome pie. GNOME Shell as of version 3.16 is by default making use of "symbolic"-named icon for the active app icon in the top bar, and this icon might have been added for that purpose, cf. also https://github.com/numixproject/numix-icon-theme/issues/485.

**Comment by bil-elmoussaoui** [comment]
It's not a bug as you said i just tried using the default gnome shell theme and as you said this icon is used in the top bar too

**Comment by dirtydancing** [comment]
Ah ok, so this is the second icon then after `gnome-tweak-tool-symbolic.svg` that is located in the fallback icon theme `hicolor`. I will make a notice to this in https://github.com/numixproject/numix-icon-theme/issues/485.

**Comment by wa4557** [comment]
Does this icon appear in the tray/panel? (Like for example the battery or sound icons)

**Comment by bil-elmoussaoui** [comment]
Yes it's used in both tray/panel and for gnome 3.16 top bar

**Comment by wa4557** [comment]
Well than I suppose this is a wontfix, I think there shouldn't be `symbolic` icons in `status`

**Comment by bil-elmoussaoui** [comment]
It's not in status, the icon is in ```scalable```

**Comment by Foggalong** [comment]
It's actually the other way around. GNOME are trying to force program developers to using `symbolic` type icons for their `status` icons.

**Comment by wa4557** [comment]
Does that mean that we should/could add a `gnome-pie-symbolic.svg` in `scalable`?

**Comment by Foggalong** [comment]
In theory, yes.

The problem is that the GNOME devs are basically forcing icon creators to make a scalable version of every app icon, even those that don't have a normal status/panel bar icon. These icons will then be shown in the top left corner where GNOME Shell puts the app menu.

These will need to be stored in `/scalable/apps/`because they are in theory needed for every single app and are scalable icons. But then also app developers have been previously told by GNOME to rename their status/panel icons to `$ICON-symbolic.svg`. Because keeping compatibility with DEs that don't use `*-symbolic.svg` icons by default in the panel (like XFCE, Cinnamon, etc) is a must, you then also have to stick icons with `*-symbolic.svg` names in the fixed size `status` directories. 

Now you have this horrible mess of clashing icons of various sizes and all the same names. It's not hard to see why some icon theme developers just target one particular DE and then tell everyone else to shove it.

**Comment by dirtydancing** [comment]
Well, at the moment I cannot say that I fully understand what is going on in GNOME 3.16:

1) Because this thread started specifically with Gnome Pie: the original icon I get with current version 0.6.2 is entirely different. I guess the icon above is outdated. Which version of Gnome Pie are you using @bil-elmoussaoui?

![gnome-pie](https://cloud.githubusercontent.com/assets/7164227/7867590/48cea95e-0577-11e5-8b1c-2a02cf393934.png)

2) The "symbolic" Gnome Pie icon is showing in the upper left top bar as active app icon, 

![gnome-pie_2](https://cloud.githubusercontent.com/assets/7164227/7867646/998f843a-0577-11e5-8496-1aacda6a30ac.png)

but not in the "legacy" status tray, be it in the default lower left or in the upper right top bar with the TopIcons extension (Arch/GNOME Shell 3.16.2), although I can right-click the area and get to a Gnome Pie menu.

![gnome-pie_3](https://cloud.githubusercontent.com/assets/7164227/7867751/2ab684ae-0578-11e5-8a69-755403d5a144.png)

3) Also other "legacy" status icons are not showing as Numix/Numix-Circle icons, but rather as the original icons. So I checked with `redshift-status-on.svg`: copying that from a fixed size status directory to `scalable/apps` resulted in the Numix status icon being displayed in the "legacy" status tray. Strangely enough, this went for "on" and "off", although only "on" was in `scalable/apps`.

4) This effect seems to imply that the status icon does not have to be "symbolic"-named, but has to be in `scalable/apps`, no matter what its name is.

5) This effect might give us a clue for the infamous status icon issue over at https://github.com/numixproject/numix-icon-theme/issues/190 and why this is showing different results with GTK 3.16: It might not be the icon lookup mechanism that has changed with GTK 3.16, but rather the order of the directories that are looked at.

Again strangely enough, I seem to get somewhat different results depending on the GS theme (again, not sure what is going on, so I will not go into further detail at this point), so I am going with the default Adwaita GS theme for now.

While I agree that this entire thing currently looks like a mess, at least this might have provided a clue about why things seem to have changed with GTK 3.16 in relation to https://github.com/numixproject/numix-icon-theme/issues/190.

Then again, all of this might only apply to GNOME Shell, so even when e.g. Ubuntu/Unity switches to GTK 3.16, there might still be Numix Circle icons in the top bar being displayed as status icons.

**Comment by Foggalong** [comment]
Unity will be switching to Qt5 at some point down the line which is going to throw a whole new series of spanners in the works.

**Comment by dirtydancing** [comment]
Yeah, looks like this will be "dynamic" for some time to come :) To me, one key to be able to navigate this is to have different DE in active use, so issues can be fixed with the least possible breakage. This might also lead to a situation when a universal approach is not possible anymore, but currently things are OK.

As to GNOME Shell 3.16, by default the "legacy" status icons are not visible, but rather stored in the lower left and can be drawn out. So the situation is not too bad after all. I actually appreciate how much GNOME Shell gets out of the way by default. It will be interesting to see if other reports will come up.

**Comment by wa4557** [comment]
Going through some old issues (it would be really cool to finish all issues; except the list ones maybe): I could easily make icons for gnome-pie (but probably next week when I have time again), but I don't know where to put them. @bil-elmoussaoui Where is the icon located in GNOME? @dirtydancing (If you're still around) any suggestions?

**Comment by bil-elmoussaoui** [comment]
The icon is in the scalable folder `/scalable/apps/`

**Comment by dirtydancing** [comment]
Well, if we are talking about the monochrome 16x16 sized `gnome-pie-symbolic.svg`, we are again at the issue of https://github.com/numixproject/numix-icon-theme/issues/485. So far, there are no Numix active app icons for the GNOME Shell top bar. If such app `*-symbolic.svg`-icons exist, they are picked up from the fallback icon themes due to the inheritances.

`gnome-pie.svg` in contrast is a normal 48x48 sized icon in Numix-Circle.

**Comment by wa4557** [comment]
ok so a wontfix then

**Comment by dirtydancing** [comment]
Yes, for `gnome-pie-symbolic.svg` I would agree that for now this is a won't fix.

**Comment by palob** [comment]
Yes, wontfix, we don't need to replicate symbolic icons. There is no room for a distinct Numix design.
